public/src/components/channelManagement/headerTests/headerTestEditor.tsx

```diff
@@ -6,69 +6,40 @@ import { DeviceType, setStatus, UserCohort } from '../helpers/shared';
 import { Typography } from '@material-ui/core';
 import HeaderTestVariantEditor from './headerTestVariantEditor';
 import TestVariantsEditor from '../testVariantsEditor';
-import TestEditorHeader from '../testEditorHeader';
 
 import TestEditorTargetAudienceSelector from '../testEditorTargetAudienceSelector';
 
-import TestEditorActionButtons from '../testEditorActionButtons';
 import LiveSwitch from '../../shared/liveSwitch';
-import useValidation from '../hooks/useValidation';
 import { HeaderTest, HeaderVariant } from '../../../models/header';
 import { getDefaultVariant } from './utils/defaults';
 import TestEditorVariantSummary from '../testEditorVariantSummary';
 
 import { ControlProportionSettings } from '../helpers/controlProportionSettings';
 import TestVariantsSplitEditor from '../testVariantsSplitEditor';
 import { useStyles } from '../helpers/testEditorStyles';
+import { ValidatedTestEditorProps } from '../validatedTestEditor';
 
-interface HeaderTestEditorProps {
-  test: HeaderTest;
-  onChange: (updatedTest: HeaderTest) => void;
-  onValidationChange: (isValid: boolean) => void;
-  visible: boolean;
-  editMode: boolean;
-  onDelete: () => void;
-  onArchive: () => void;
-  onTestSelected: (testName: string) => void;
-  testNames: string[];
-  testNicknames: string[];
-  createTest: (newTest: HeaderTest) => void;
-}
-
-const HeaderTestEditor: React.FC<HeaderTestEditorProps> = ({
+const HeaderTestEditor: React.FC<ValidatedTestEditorProps<HeaderTest>> = ({
```

Comment:
Most of this file hasn't changed. It now takes the standard `ValidatedTestEditorProps`